epoch/api/channels_api_usage.md

```diff
@@ -78,19 +78,14 @@ in the chain, and the others are metadata used for the connection itself.
   | ttl | 1000 |
 
   The `initiator` will be connecting to the `responder` on localhost:1234
-  We will be using following tools
-
-  * [wscat](https://github.com/websockets/wscat)
-  * [unpack](https://github.com/aeternity/aepp-sdk-js/blob/develop/examples/unpack.js)
-  * [sign](https://github.com/aeternity/aepp-sdk-js/blob/develop/examples/sign.js)
```

Comment:
It is dangling

Reply:
I believe these utils could be used in following way.

`npm install -g @aeternity/aepp-sdk`

and then

`aecrypto unpack tx$...`

`aecrypto sign tx$...`

However binaries are not defined in [package.json](https://github.com/aeternity/aepp-sdk-js/blob/develop/package.json). What do you think @e-user?